packages/commons/src/fixtures/forms.ts

```diff
@@ -1224,7 +1224,8 @@ export const TENNIS_CLUB_DECLARATION_FORM = defineDeclarationForm({
             id: 'event.tennis-club-membership.action.declare.form.section.recommender.field.id.label'
           }
         }
-      ]
+      ],
+      conditional: not(never())
```

Comment:
Let's just add a comment here that states this is only present for testing page conditionals